src/server/lib/templateRules.json

```diff
@@ -1255,6 +1255,32 @@
       "ecCodes": [
         "1.11"
       ]
+    },
+    "Cancellation_Reason__c": {
+      "key": "Cancellation_Reason__c",
+      "index": 47,
+      "required": "Conditional",
```

Comment:
might be interesting to implement this conditional require in the code. i always assumed we'd (in `validation-rules.js` method `generateRules`) would set `required` to a function, something like:

```javascript
rules.ec1.Cancellation_Reason__C.required = (record) => record.project_status_tbd === 'Cancelled'
```

Reply:
It would be good. There's a lot of the conditional logic already embedded within the input template as conditional formatting. While this is only a visual treatment within Excel, it might offer you a template to pass information to this kind of function.